.github/workflows/run-test.yml

```diff
@@ -86,13 +86,23 @@ jobs:
 
       - name: Run tests on Firebase Test Lab
         run: |-
-          gcloud firebase test android run ".github/tests.yml:android-pixel-2" --results-dir="${{ steps.generate-dir.outputs.results_dir }}" --results-bucket="${{ steps.generate-dir.outputs.bucket }}"
+          gcloud firebase test android run \
```

Comment:
**info**: instead of using `.github/tests.yml` I extracted all this setup directly to the command execution because it's easier to have everything in one file ;)